docs/bootstrap/README.md

```diff
@@ -2,7 +2,12 @@
 
 1) [Sign up to DigitalOcean](https://www.digitalocean.com/?refcode=2962aa9e56a1&utm_campaign=Referral_Invite&utm_medium=Referral_Program&utm_source=CopyPaste)
 2) [Download Terraform](https://www.terraform.io)
-3) Clone this gist using the URL from the address bar
+3) Download the [cloud-config.tpl](cloud-config.tpl) and [main.tf](main.tf) files.
+   Or directly from the terminal using `wget`:
+   ```bash
+   $ wget --quiet https://raw.githubusercontent.com/openfaas/faasd/master/docs/bootstrap/main.tf
```

Comment:
Can you use curl here? wget isn't on MacOS by default

`curl -SLso FILENAME https://`